src/main/java/org/jabref/logic/exporter/ExporterFactory.java

```diff
@@ -55,6 +55,7 @@ public static ExporterFactory create(PreferencesService preferencesService,
         exporters.add(new TemplateExporter("MIS Quarterly", "misq", "misq", "misq", StandardFileType.RTF, layoutPreferences, saveOrder));
         exporters.add(new TemplateExporter("CSL YAML", "yaml", "yaml", null, StandardFileType.YAML, layoutPreferences, saveOrder, BlankLineBehaviour.DELETE_BLANKS));
         exporters.add(new TemplateExporter("Hayagriva YAML", "hayagrivayaml", "hayagrivayaml", null, StandardFileType.YAML, layoutPreferences, saveOrder, BlankLineBehaviour.DELETE_BLANKS));
+        exporters.add(new TemplateExporter("Endnote XML", "endnotexml", "Endnote XML", "endnotexml", StandardFileType.XML, layoutPreferences, saveOrder));
```

Comment:
A Template exporter is one that only uses the .layout files -> Remove this line

Reply:
Done.